docs/user-guide/configure.md

```diff
@@ -74,9 +65,18 @@ You can add any number of keys in the object. For example, you can:
 }
 ```
 
+Some rules and options accept regex. You can enforce these common cases:
```

Comment:
[note] Pulled from the using regex page as it's better suited here where it's contextual.